src/components/CardSeparator.tsx

```diff
@@ -10,7 +11,27 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-type Color = 'accent' | 'primary' | 'secondary' | 'tertiary' | 'transparent'
+type Gradient =
+  | 'secondary-to-transparent'
+  | 'accent-to-secondary'
+  | 'accent-to-transparent'
+interface CardSeparatorProps {
+  number: number
+  gradient: Gradient
+  vertical?: boolean
+  customHeight?: THeight
+}
+
+function getTailwindClassnames(gradient: Gradient) {
```

Comment:
Use `classnames` here or whatever function needed from `tailwind-classnames`

Reply:
Also, name of this function implies that you're getting all the tailwind classnames, you should specify which classnames you're getting here